docs/source/deployment/CORS_Headers.rst

```diff
@@ -0,0 +1,95 @@
+========================================
+Add CORS Headers to MOTECH Configuration
+========================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Add CORS support to Apache server
+=================================
```

Comment:
I don't get what is the set up here. I am assuming we have a webapp and the Motech application? Is the webapp running on Apache and Motech on Tomcat/Jetty? Or is Motech running on Tomcat/Jetty behind Apache?